src/de.rs

```diff
@@ -33,6 +33,37 @@ pub fn from_slice<'de, T>(bytes: &'de [u8]) -> Result<T, Error>
 ///
 /// This function will attempt to interpret `s` as a TOML document and
 /// deserialize `T` from the document.
+///
+/// # Examples
+///
+/// ```no_run
```

Comment:
Uh... No, you are right!  I just added it **without thinking** because someone asked me to add it in one other PR: https://github.com/rust-lang/rust/pull/40904/files/5a7e9ad64a160c9ebfb35969f2bdd1065bc154bb#diff-12dfbaa7f22fc4118fe031d0967e78e6

Still I'm not sure I need to remove it or leave it.